darwin/options.py

```diff
@@ -148,6 +149,27 @@ def __init__(self):
         )
         parser_split.add_argument("-s", "--seed", type=int, required=False, default=0, help="Split seed.")
 
+        # File listing
+        parser_files = dataset_action.add_parser("files", help="Lists file in a remote dataset")
+        parser_files.add_argument(
+            "dataset",
+            type=str,
+            help="[Remote] Dataset name: to list all the existing dataset, run 'darwin dataset remote'. ",
+        )
+        parser_files.add_argument("--only-filenames", action="store_true", help="Only prints out filenames")
+        parser_files.add_argument("--status", type=str, required=False, help="Comma separated list of statuses")
```

Comment:
A minor thing, but why not making this flag `nargs="+"` instead of a comma-separated list? I think it might be more intuitive when using it in the command line.

A second minor thing: "comma separated" -> "comma-separated"